### PR TITLE
Sprint 50

### PR DESCRIPTION
**Pull request by M-Francia**
## Description

Added Hedera Wallet Connect.
Now, the token address is retrieved from the mirror after creating a stablecoin, avoiding extra signatures and fees.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue) 🐞
- [x] New feature (non-breaking change which adds functionality) ✨
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 💥
- [ ] This change requires a documentation update 📖

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. 🧪

- [x] Local Tests (npm run test)
- [ ] Local Example (npx tsx examples/index.ts)
- [ ] Local Github Actions (act pull_request)

**Test Configuration**:

- Node version (latest LTS):
  - [ ] 16
  - [x] 18
  - [ ] 20

### CODE REVIEW

File: .github/workflows/all.testWithRpc.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   test-node:
     name: testing
-    runs-on: token-studio-linux-medium
+    runs-on: ubuntu-latest
```
Comment (nathanklick) [inline .github/workflows/all.testWithRpc.yml:9]:
```suggestion
    runs-on: token-studio-linux-medium
```

Comment (M-Francia) [reply .github/workflows/all.testWithRpc.yml:9]: @nathanklick with that change we have problems to build the sdk..

Comment (rbarkerSL) [reply .github/workflows/all.testWithRpc.yml:9]: That is a required change that has been made at the organization level. Please revert this change and correct any subsequent build issues.

**Review by MiguelLZPF — DISMISSED** [review]

File: web/package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-	"version": "1.24.0",
+	"version": "1.25.0",
 	"files": [
 		"build/"
 	],
 	"scripts": {
 		"start": "react-app-rewired start",
 		"build": "react-app-rewired build",
-		"test": "react-app-rewired test --maxWorkers=50% --watchAll=false",
+		"test": "react-app-rewired test --maxWorkers=25% --watchAll=false --runInBand --cache=false",
```
Comment (rbarkerSL) [inline web/package.json:10]: max workers and runInBand seem to be incompatible with each other.

From the checks: 
[Both --runInBand and --maxWorkers were specified, but these two options do not make sense together. Which is it?](https://github.com/hashgraph/stablecoin-studio/actions/runs/10813475080/job/29997591581?pr=1180#:~:text=Both%20%2D%2DrunInBand%20and%20%2D%2DmaxWorkers%20were%20specified%2C%20but%20these%20two%20options%20do%20not%20make%20sense%20together.%20Which%20is%20it%3F)

I recommend removing the `--runInBand` argument here.

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

File: .github/workflows/all.publish.yml
Comment (rbarkerSL) [inline .github/workflows/all.publish.yml:1]: Need to add the step-security hardened runner step on each job that utilizes steps in the workflow

## Hardened Runner Step
```yaml
      - name: Harden Runner
        uses: step-security/harden-runner@f086349bfa2bd1361f7909c78558e816508cdc10 # v2.8.0
        with:
          egress-policy: audit
```

Affected jobs in `all.publish.yml`:

- publish

File: .github/workflows/all.testWithRpc.yml
Comment (rbarkerSL) [inline .github/workflows/all.testWithRpc.yml:1]: Need to add the step-security hardened runner step on each job that utilizes steps in the workflow

## Hardened Runner Step
```yaml
      - name: Harden Runner
        uses: step-security/harden-runner@f086349bfa2bd1361f7909c78558e816508cdc10 # v2.8.0
        with:
          egress-policy: audit
```

Affected jobs in `all.testWithRpc.yml`:

- test-node

**Review by rbarkerSL — CHANGES_REQUESTED** [review]

File: .github/workflows/all.testWithRpc.yml
Comment (MiguelLZPF) [reply .github/workflows/all.testWithRpc.yml:1]: Added in latest commit

File: .github/workflows/all.publish.yml
Comment (MiguelLZPF) [reply .github/workflows/all.publish.yml:1]: Added in latest commit

File: web/package.json (hunk 2 above)
Comment (MiguelLZPF) [reply web/package.json]: Thanks! It was a copy paste error 👀

**Review by MiguelLZPF — APPROVED** [review]

**Review by gregscullard — APPROVED** [review]
LGTM

**Comment by AlbertoMolinaIoBuilders** [comment]
Hello @rbarkerSL @nathanklick , could you have a look at this and see if we can merge it?

**Review by rbarkerSL — APPROVED** [review]
LGTM

File: .github/workflows/all.testWithRpc.yml (hunk 1 above)
Comment (M-Francia) [reply .github/workflows/all.testWithRpc.yml]: @nathanklick changes done =)

**Review by jeromy-cannon — APPROVED** [review]

**Pull request closed without merging**